Time/epoch.60s.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# <xbar.title>Pretty Epoch Time</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Anthony Jarvis-Clark</xbar.author>
+# <xbar.author.github>anthonyclarka2</xbar.author.github>
+# <xbar.desc>Displays epoch (unix) time separated by commas.</xbar.desc>
+# <xbar.image>http://i.imgur.com/ltLqOcy.png</xbar.image>
+# <xbar.dependencies>bash,gnu-sed</xbar.dependencies>
+
+EPOCHNOW=$(/bin/date +%s | /usr/local/bin/gsed ':a;s/\B[0-9]\{3\}\>/,&/;ta')
```

Comment:
@anthonyclarka2 

Brew installs items into a different location on M1 silicon. Any qualms with changing your script to this:

```suggestion
BREWPATH=/usr/local/bin
if [[ $(arch) == 'arm64' ]]; then
  BREWPATH=/opt/homebrew/bin
fi

EPOCHNOW=$(/bin/date +%s | ${BREWPATH}/gsed ':a;s/\B[0-9]\{3\}\>/,&/;ta')
```